File: LLM-Translations/GPT-4/codenet/Java/Go/s428436430.go
package main

import (
	"fmt"
)

type calArea struct{}

func (c *calArea) Area(w, h int) int {
	return w * h
}

func main() {
	var arr [4]int
	for i := 0; i < 4; i++ {
		fmt.Scan(&arr[i])
	}

	ab := &calArea{}
	area1 := ab.Area(arr[0], arr[1])
	area2 := ab.Area(arr[2], arr[3])

	if area1 > area2 {
		fmt.Println(area1)
	} else if area1 < area2 {
		fmt.Println(area2)
	} else if area1 == area2 {
		fmt.Println(area1)
	}

}

// 

File: LLM-Translations/GPT-4/codenet/Java/Python/s428436430.py
import sys

class CalArea:
    def area(self, w, h):
        area = w * h
        return area

if __name__ == "__main__":
    arr = [int(x) for x in sys.stdin.readline().split()]

    ab = CalArea()
    area1 = ab.area(arr[0], arr[1])
    area2 = ab.area(arr[2], arr[3])

    if area1 > area2:
        print(area1)
    elif area1 < area2:
        print(area2)
    elif area1 == area2:
        print(area1)


